transactions/shardedCollection/setup_sharded_collection.cdc

```diff
@@ -25,7 +26,7 @@ transaction(numBuckets: UInt64) {
                 acct.unlink(/public/MomentCollection)
             }
 
-            acct.link<&{TopShot.MomentCollectionPublic}>(/public/MomentCollection, target: /storage/ShardedMomentCollection)
+            acct.link<&{NonFungibleToken.Receiver, NonFungibleToken.CollectionPublic, TopShot.MomentCollectionPublic}>(/public/MomentCollection, target: /storage/ShardedMomentCollection)
```

Comment:
Why not `MetadataViews.ResolverCollection` here ?

Reply:
the sharded collection doesn't implement that interface so we can't link it